### PR TITLE
Import CA state data

### PR DESCRIPTION
**Pull request by drewsamnick**
This lets us import the CA computer science offerings. Since the raw file was so large, I added in a script to do some preprocessing on it. That brings the file down from over a million row to a few thousand (the size drops from 143M to 433K.)

### CODE REVIEW

**Comment by drewsamnick** [comment]
[Import 2016-2017 California data](https://trello.com/c/NKoukTyK/132-import-2016-2017-california-data)

File: bin/oneoff/census/state-data/preprocess-ca-data.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    filename = f
+  end
+  opts.on('-h', '--help', 'Prints this help') do
+    puts opts
+    exit
+  end
+end.parse!
+
+# The raw file we get from CA (CoursesTaught via https://www.cde.ca.gov/ds/sd/df/filesassign.asp)
+# Has data for all courses in all schools in CA, not jsut CS.
```
Comment (breville) [inline bin/oneoff/census/state-data/preprocess-ca-data.rb]: nit: "just"

Comment (drewsamnick) [reply bin/oneoff/census/state-data/preprocess-ca-data.rb]: Fixed

File: dashboard/app/models/census/state_cs_offering.rb [hunk 2]
```diff
@@ -37,8 +40,29 @@ def self.construct_state_school_id(state_code, row_hash)
     end
   end
 
+  CA_COURSE_CODES = %w(
```
Comment (breville) [inline dashboard/app/models/census/state_cs_offering.rb:43]: Do these map to anything we should comment on, or is the list good enough as is?

**Comment by drewsamnick** [comment]
I've been manually uploading the files to S3 but I think it would be good to add in some scripts to handle that. I'll put an item not he backlog for that.

File: dashboard/app/models/census/state_cs_offering.rb (hunk 2 above)
Comment (drewsamnick) [reply dashboard/app/models/census/state_cs_offering.rb:43]: This is the list of CS course codes that I got from Miranda.

**Pull request merged**